src/hooks/useNotification.ts

```
@@ -1,21 +1,54 @@
+import { EventSourcePolyfill, NativeEventSource } from 'event-source-polyfill';
 import { useEffect, useState } from 'react';
+import { getTokens } from '../store/authStore';
+import { httpClient } from '../api/http.api';
 
 const useNotification = () => {
   const [isSignal, setIsSignal] = useState<boolean>(false);
```

ESLint

[error] 'setIsSignal' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)